src/main/java/com/faforever/client/patch/GameUpdaterImpl.java

```diff
@@ -144,19 +140,18 @@ private CompletableFuture<Void> downloadMissingSimMods(Set<String> simModUids) {
         .map(modService::downloadAndInstallMod).toArray(CompletableFuture[]::new));
   }
 
-  private CompletableFuture<PatchResult> updateFeaturedMod(FeaturedModBean featuredMod, Integer version) {
-    for (FeaturedModUpdater featuredModUpdater : featuredModUpdaters) {
-      if (featuredModUpdater.canUpdate(featuredMod)) {
-        return featuredModUpdater.updateMod(featuredMod, version);
-      }
-    }
-    throw new UnsupportedOperationException("No updater available for featured mod: " + featuredMod
-        + " with version:" + version);
+  private CompletableFuture<PatchResult> updateFeaturedMod(FeaturedModBean featuredMod, Integer version, boolean forReplays) {
+    return featuredModUpdaters.stream()
+        .findFirst()
+        .map(updater -> updater.updateMod(featuredMod, version, forReplays))
+        .orElseThrow(() -> new UnsupportedOperationException("No updater available for featured mod: " + featuredMod
+            + " with version: " + version));
   }
```

Comment:
Without the canUpdate method this doesn't make much sense. So either add back in the canUpdate to make it clear there is the possibility that a featuredModUpdater can not update a featured mod

Reply:
It seems like the second half of the sentence is missing